tests/unit_tests/job_runner/test_file_reporter.py

```diff
@@ -111,7 +110,7 @@ def test_report_with_failed_exit_message_argument(reporter):
             "<reason>massive_failure</reason>" in content
         ), "ERROR file missing reason"
         assert (
-            "<stderr: Not redirected>" in content
+            "<stderr>\nstderr: Not redirected\n</stderr>" in content
```

Comment:
Would "`<stderr>Not redirected</stderr`>" look even better?

Reply:
Yes, fully agree, but all the stderr needs to be removed :) Can do of-course :+1: )